src/core/configs/cluster.rs

```diff
@@ -287,6 +287,9 @@ impl Cluster {
                 debug!("cannot find node: {} in cluster config", node.name());
                 format!("cannot find node: {} in cluster config", node.name())
             })?;
+        if self.vdisks().iter().any(|vdisk| vdisk.replicas().len() < node.quorum()) {
+            return Err(String::from("Quorum is more than amount of replicas for at least one VDisk"));
```

Comment:
`vdisk` name should be specified inside the error description. This will make it easier to find and fix the problem

Reply:
Done in new commit!